### PR TITLE
refactor!: align `PasscodeType` to `MessageTypes`

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

 
remove database `PasscodeType` and use `MessageTypes` as SSOT

we'll change `MessageTypes` to `VerificationCodeType` in the following PR

## Testing

CI

### CODE REVIEW

File: packages/core/src/routes/interaction/utils/passcode-validation.ts [hunk 1]
```diff
@@ -8,29 +8,29 @@ import type { SendPasscodePayload, PasscodeIdentifierPayload } from '../types/in
...
+  eventToMessageTypesMap[event];
 
 export const sendPasscodeToIdentifier = async (
   payload: SendPasscodePayload,
   jti: string,
   createLog: LogContext['createLog']
 ) => {
   const { event, ...identifier } = payload;
-  const passcodeType = getPasscodeTypeByEvent(event);
+  const MessageTypes = getMessageTypesByEvent(event);
```
Comment (darcyYe) [inline packages/core/src/routes/interaction/utils/passcode-validation.ts:28]:
```suggestion
  const messageTypes = getMessageTypesByEvent(event);
```

File: packages/core/src/routes/interaction/utils/passcode-validation.ts [hunk 2]
```diff
@@ -8,29 +8,29 @@ import type { SendPasscodePayload, PasscodeIdentifierPayload } from '../types/in
...
 
   const log = createLog(`Interaction.${event}.Identifier.VerificationCode.Create`);
   log.append(identifier);
 
-  const passcode = await createPasscode(jti, passcodeType, identifier);
+  const passcode = await createPasscode(jti, MessageTypes, identifier);
```
Comment (darcyYe) [inline packages/core/src/routes/interaction/utils/passcode-validation.ts:33]:
```suggestion
  const passcode = await createPasscode(jti, messageTypes, identifier);
```

File: packages/core/src/routes/interaction/utils/passcode-validation.ts [hunk 3]
```diff
@@ -42,11 +42,11 @@ export const verifyIdentifierByPasscode = async (
   createLog: LogContext['createLog']
 ) => {
   const { event, passcode, ...identifier } = payload;
-  const passcodeType = getPasscodeTypeByEvent(event);
+  const MessageTypes = getMessageTypesByEvent(event);
```
Comment (darcyYe) [inline packages/core/src/routes/interaction/utils/passcode-validation.ts:45]:
```suggestion
  const messageTypes = getMessageTypesByEvent(event);
```

File: packages/core/src/routes/interaction/utils/passcode-validation.ts [hunk 4]
```diff
@@ -42,11 +42,11 @@ export const verifyIdentifierByPasscode = async (
...
 
   // TODO: @Simeng maybe we should just log all interaction payload in every request?
   const log = createLog(`Interaction.${event}.Identifier.VerificationCode.Submit`);
   log.append(identifier);
 
-  await verifyPasscode(jti, passcodeType, passcode, identifier);
+  await verifyPasscode(jti, MessageTypes, passcode, identifier);
```
Comment (darcyYe) [inline packages/core/src/routes/interaction/utils/passcode-validation.ts:51]:
```suggestion
  await verifyPasscode(jti, messageTypes, passcode, identifier);
```

File: packages/core/src/routes/interaction/utils/passcode-validation.ts (hunk 2 above)
Comment (darcyYe) [inline packages/core/src/routes/interaction/utils/passcode-validation.ts:33]:
```suggestion
  const passcode = await createPasscode(jti, messageTypes, identifier);
```

Comment (darcyYe) [inline packages/core/src/routes/interaction/utils/passcode-validation.ts:33]:
```suggestion
  const passcode = await createPasscode(jti, messageTypes, identifier);
```

**Review by darcyYe — COMMENTED** [review]
IT failed, should take a look.

**Comment by gao-sun** [comment]
@darcyYe  i think rerun can solve it. @charIeszhao it working on the stable version

**Review by darcyYe — APPROVED** [review]

**Comment by charIeszhao** [comment]
> @darcyYe i think rerun can solve it. @charIeszhao it working on the stable version

Yeah, I'm still trying but unfortunately I find it hard to reproduce it. Will keep trying.

**Review by wangsijie — APPROVED** [review]

**Pull request merged**